fil-proofs-tooling/src/bin/benchy/window_post.rs

```diff
@@ -105,49 +117,67 @@ pub fn run_window_post_bench<Tree: 'static + MerkleTreeTrait>(
     let cache_dir = tempfile::tempdir().unwrap();
     let sector_id = SectorId::from(SECTOR_ID);
 
-    let phase1_output = seal_pre_commit_phase1::<_, _, _, Tree>(
-        porep_config,
-        cache_dir.path(),
-        staged_file.path(),
-        sealed_file.path(),
-        PROVER_ID,
-        sector_id,
-        TICKET_BYTES,
-        &piece_infos,
-    )?;
-
-    validate_cache_for_precommit_phase2::<_, _, Tree>(
-        cache_dir.path(),
-        sealed_file.path(),
-        &phase1_output,
-    )?;
-
-    let seal_pre_commit_output = seal_pre_commit_phase2::<_, _, Tree>(
-        porep_config,
-        phase1_output,
-        cache_dir.path(),
-        sealed_file.path(),
-    )?;
+    let seal_pre_commit_phase1_measurement = measure(|| {
+        seal_pre_commit_phase1::<_, _, _, Tree>(
+            porep_config,
+            cache_dir.path(),
+            staged_file.path(),
+            sealed_file.path(),
+            PROVER_ID,
+            sector_id,
+            TICKET_BYTES,
+            &piece_infos,
+        )
+    })
+    .expect("failed in seal_pre_commit_phase1");
```

Comment:
This function returns already an anyhow result, hence you could use `.context()?` instead of `.expect()` which then would return an error instead of panicking. The applies to all uses of `.expect()`.